### PR TITLE
Localhost and storybook AV support

### LINKED ISSUE #3931: Enable AV embed playback on local and storybook
**Is your feature request related to a problem? Please describe.**
Enable media player for local development and storybook stories. 

**Describe the solution you'd like**
The following URLs should have real AV playback. 

**Live radio**
http://localhost.bbc.com:7080/korean/bbc_korean_radio/liveradio.amp
http://localhost.bbc.com:7080/korean/bbc_korean_radio/liveradio
`npm run build && npm run start`

**Articles**
http://localhost.bbc.com:7080/news/articles/c3wmq4d1y3wo.amp
http://localhost.bbc.com:7080/news/articles/c3wmq4d1y3wo

**Storybook**
http://localhost.bbc.com:9001/?path=/story/containers-media-player-canonical--default
http://localhost.bbc.com:9001/?path=/story/containers-media-player-amp--default
http://localhost.bbc.com:9001/?path=/story/pages-media-page--default
`npm run storybook`

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Manual testing that AV playback works as expected on the above URLs. 

- [x] This feature is expected to need manual testing.

**Additional context**
Also check the simorgh stories.

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #3931

**Overall change:** Extends the `embedUrl` helper to support multiple types. Changes the live radio `src` to be absolute not relative. 

**Code changes:**

- Extends the `embedUrl` helper to have multiple types and also incorporate the `vpid` for articles into the `assetId` prop
- Uses the `embedUrl` in `MediaPageBlocks/LiveRadio/index.jsx` and now passes in the `origin`
- Adds is `isAmp` logic to the `MediaPageBlocks/LiveRadio/index.jsx`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

Testing notes on the issue.

### CODE REVIEW

File: src/app/containers/MediaPlayer/helpers/embedUrl.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ const buildBaseUrl = origin => {
   return `https://www.test.bbc${tld}`;
 };
 
-const embedUrl = ({ origin, assetId, vpid, isAmp = false }) => {
-  const avRoute = 'ws/av-embeds/articles';
+const embedUrl = ({ origin, type, assetId, isAmp = false }) => {
```
Comment (simonsinclair) [inline src/app/containers/MediaPlayer/helpers/embedUrl.js]: Could we rename `assetId` to something like `requestUrl` now that `assetId` and `vpid` are combined?

File: src/app/containers/MediaPlayer/helpers/embedUrl.test.js [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
-const assetId = 'foo';
-const vpid = 'bar';
+const assetId = 'foo/bar';
+const type = 'articles';
 
 const testCases = [
   {
     description: 'CANONICAL: builds a URL for LIVE environment on .co.uk',
-    expected: `https://www.bbc.co.uk/ws/av-embeds/articles/${assetId}/${vpid}`,
+    expected: `https://www.bbc.co.uk/ws/av-embeds/articles/${assetId}`,
```
Comment (simonsinclair) [inline src/app/containers/MediaPlayer/helpers/embedUrl.test.js]: Could we have a test that shows a new `type`? e.g. media

**Review by simonsinclair — CHANGES_REQUESTED** [review]
I peer reviewed this with Phil this afternoon (thanks, Phil) and discussed some changes. Please see annotations.

**Comment by pjlee11** [comment]
image

**Review by dr3 — APPROVED** [review]

**Review by ryanmccombe — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]
The changes look great and the new tests pass. 👍

**Comment by paruchurisilpa** [comment]
looks good to me..

**Comment by chris-hinds** [comment]
Admin merging as chromatic build has passed but the status has not updated

image

**Pull request merged**